ui/perfherder/alerts/AlertHeader.jsx

```diff
@@ -40,69 +50,77 @@ const AlertHeader = ({ alertSummary, repoModel, issueTrackers }) => {
-              href={repoModel.getPushLogRangeHref({
-                fromchange: alertSummary.prev_push_revision,
-                tochange: alertSummary.revision,
-              })}
-              target="_blank"
-              rel="noopener noreferrer"
+      <Container>
+        <Row className="font-weight-normal">
+          <span className="align-middle">{`${moment(
+            alertSummary.push_timestamp * 1000,
+          ).format('ddd MMM D, HH:mm:ss')} · `}</span>
+          <UncontrolledDropdown tag="span">
+            <DropdownToggle
+              className="btn-link text-info p-0"
+              color="transparent"
+              caret
             >
-              <DropdownItem>Pushlog</DropdownItem>
-            </a>
-          </DropdownMenu>
-        </UncontrolledDropdown>
-        {bugNumber && (
-          <span>
-            <span className="align-middle"> · </span>
-            {alertSummary.issue_tracker && issueTrackers.length > 0 ? (
+              {alertSummary.revision.slice(0, 12)}
+            </DropdownToggle>
+            <DropdownMenu>
+              <a
+                className="text-dark"
+                href={getJobsUrl({
+                  repo: alertSummary.repository,
+                  fromchange: alertSummary.prev_push_revision,
+                  tochange: alertSummary.revision,
+                })}
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <DropdownItem>Jobs</DropdownItem>
+              </a>
               <a
-                className="text-info align-middle"
-                href={getIssueTrackerUrl(alertSummary)}
+                className="text-dark"
+                href={repoModel.getPushLogRangeHref({
+                  fromchange: alertSummary.prev_push_revision,
+                  tochange: alertSummary.revision,
+                })}
                 target="_blank"
                 rel="noopener noreferrer"
               >
-                {bugNumber}
+                <DropdownItem>Pushlog</DropdownItem>
               </a>
-            ) : (
-              { bugNumber }
-            )}
-          </span>
-        )}
-      </span>
+            </DropdownMenu>
+          </UncontrolledDropdown>
+          {bugNumber && (
+            <span>
+              <span className="align-middle"> · </span>
+              {alertSummary.issue_tracker && issueTrackers.length > 0 ? (
+                <a
+                  className="text-info align-middle"
+                  href={getIssueTrackerUrl(alertSummary)}
+                  target="_blank"
+                  rel="noopener noreferrer"
+                >
+                  {bugNumber}
+                </a>
+              ) : (
+                { bugNumber }
+              )}
+            </span>
+          )}
+          <Assignee
+            assigneeUsername={alertSummary.assignee_username}
+            onAssigneeChange={updateAssignee}
+            user={user}
+          />
```

Comment:
Is this the only thing that was added (besides the code being pushed down)?

Reply:
Pretty much, yes. We're only tagging the owner on each alert summary.